src/stores/SettingsStore.js

```diff
@@ -58,6 +58,8 @@ const SettingsModel = types
     // showScore: types.optional(types.boolean, false),
 
     preserveSelectedTool: types.optional(types.boolean, true),
+
+    enableSmoothing:types.optional(types.boolean, true),
```

Comment:
spacing 

```js
enableSmoothing: types
```

Reply:
@bmartel here I'm using the pattern of this document

Reply:
what do you mean by pattern?
we actually missed this spacing part in our linter, but here should be a space. and we definitely should cover this by linting rules.

Reply:
so could you please add spaces here and in `if`s in `ImageView`?

Reply:
done!